book/forms.py

```diff
@@ -0,0 +1,7 @@
+from django import forms
+from .models import Book
+
+class BookForm(forms.ModelForm):
+    class Meta:
+        model = Book
+        fields = '__all__'
```

Comment:
nit: better to pass all field names explicitly to avoid problems with future form changes